js/src/apbct-public--2--public.js

```diff
@@ -411,6 +406,8 @@ function getJavascriptClientData() {
 	// Parse JSON properties to prevent double JSON encoding
 	resultDataJson = decodeJSONEncodedProperties(resultDataJson);
 
+	console.log(resultDataJson);
```

Comment:
need to remove?

Reply:
https://github.com/CleanTalk/wordpress-antispam/pull/164/commits/61b56ccfa4c4f1c855dae4f46fbcc02c19e480bb